packages/neutrino/bin/start.js

```diff
@@ -2,16 +2,33 @@ const ora = require('ora');
 const debounce = require('lodash.debounce');
 const { start } = require('../src');
 const base = require('./base');
+const DashboardPlugin = require("webpack-dashboard/plugin");
+const { spawn } = require('child_process');
 
 module.exports = (middleware, args) => {
   const spinner = ora({ text: 'Building project' });
+  const argsClone = Object.assign({}, args);
```

Comment:
Do we need this to be cloned rather than just overriding the value?

```js
if (args.dashboard) {
  args.quiet = true;
 // ...
}
```

Reply:
The linter complains when reassigning a value of an argument.